ui/src/components/DeviceInfo.vue

```diff
@@ -4,13 +4,13 @@
       <span class="q-mr-sm">{{ response.data.device_name }}</span>
       <q-chip
         :color="response.data.is_online ? 'green' : 'red'"
-        text-color="white"
+        text-color="accent"
```

Comment:
This change may not have been necessary. I was just doing a search and replacing default colours. 

The idea is to avoid using 'white', 'black' or other set colours, otherwise when people change the colour scheme things could disappear. For example, if the header text was 'black', and then someone changed the header colour to black, the text would vanish. So now the header text is 'secondary' referring to the secondary colour from the colour palette, which is then more likely to show up as it will offset the header colour someone chooses. 

In this instance though of the q-chip, perhaps the background colour is always the same? So a 'white' wouldn't have been a problem. Can change it back if need be.